docs/installation/ca/installing-ca-with-ldaps-connection.adoc

```diff
@@ -8,11 +8,9 @@ Follow this process to install a CA subsystem with a secure database connection.
 Prior to installation, please ensure that the link:../others/installation-prerequisites.adoc[Installation Prerequisites] are configured.
 
 == DS Configuration 
-Once the prerequisites listed above are completed, enable the SSL connection with a self-signed signing certificate as described in
-link:../others/enabling-ssl-connection-in-ds.adoc#enabling-ssl-connection[Enabling SSL Connection].
-
-Then export the signing certificate into `ds_signing.crt` as described in
-link:../others/exporting-ds-certificates.adoc#exporting-ds-signing-certificate[Exporting DS Signing Certificate].
+Once the prerequisites listed above are completed, if you had chosen to use the temporary DS bootstrap certificates during DS instance creation,
```

Comment:
Do we still need to mention "temporary", or is it already implied by "bootstrap"?

Reply:
Good catch.  Will fix.  Thanks.